package.json

```diff
@@ -23,7 +23,8 @@
     "eslint": "eslint --ext .js,.jsx,.ts,.tsx packages/",
     "type-check": "tsc --noEmit",
     "prettier-check": "yarn prettier --check 'packages/**/*.{ts,tsx,js,jsx,md}'",
-    "prettier-write": "yarn prettier --write 'packages/**/*.{ts,tsx,js,jsx,md}'"
+    "prettier-write": "yarn prettier --write 'packages/**/*.{ts,tsx,js,jsx,md}'",
+    "nop": "exit 0"
```

Comment:
Can you add a comment to this - someone coming by this in the future will be very confused :)

Reply:
Sure.

It is in the commit message, so a `git blame` would tell the story at least.